common/components/Header/components/TabsOptions.jsx

```diff
@@ -23,7 +23,8 @@ const tabs = [
         link: 'view-wallet'
     },
     {
-        name: 'Help'
+        name: 'Help',
```

Comment:
Off the top of my head each 'view' or page or whatever should have a name (Help Me) slug (help-me) which is the url, title (SEO title tag + title shown on page itself), description (OMG wat more SEO?) And ability to hide / show in the nav for different frontends (eg website vs chrome extension) - although this last one should maybe be done on the compile level? 

The ability for me to quickly turn on or off access to a view has come in super handy more than once, and having a single location that controls what views are available for which frontend makes keeping the chrome extension up to date essentially mindless.  

While in the future I hope we don't have 'turn it off and then we can figure it out' moments...it's better to be prepared...especially if I happen to be the only one around.